test/e2e/streaming_e2e_test.go

```diff
@@ -0,0 +1,184 @@
+package e2e
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"net/http"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/livepeer/go-livepeer/eth"
+	lpTypes "github.com/livepeer/go-livepeer/eth/types"
+	"github.com/livepeer/go-livepeer/pm"
+	"github.com/stretchr/testify/require"
+)
+
+func TestE2EStreamingWorkflow(t *testing.T) {
+	geth := setupGeth(t)
+	defer terminateGeth(t, geth)
+
+	// 1. Start and register Orchestrator
+	o := startOrchestrator(t, geth)
+	// 2. Start Broadcaster
+	b := startBroadcaster(t, geth)
```

Comment:
```suggestion
	o := startOrchestrator(t, geth)
	b := startBroadcaster(t, geth)
```

nit: I think you don't need these comments, because the code actually says the same. It's clear that you start both O and B in these lines.